generate.sh

```diff
@@ -6,41 +6,44 @@ rm -rf client
-cat openapi.json | \
-jq 'del(.components.schemas | (
-    .changeRequestsSchema,
-    .changeRequestSchema,
-    .changeRequestFeatureSchema,
-    .changeRequestEventSchema,
-    .changeRequestOneOrManyCreateSchema
-))' | \
-jq 'del(.paths | (
-."/api/admin/projects/{projectId}/change-requests",
-."/api/admin/projects/{projectId}/change-requests/config",
-."/api/admin/projects/{projectId}/change-requests/open",
-."/api/admin/projects/{projectId}/change-requests/pending",
-."/api/admin/projects/{projectId}/change-requests/pending/{featureName}",
-."/api/admin/projects/{projectId}/change-requests/{changeRequestId}/changes/{changeId}",
-."/api/admin/projects/{projectId}/change-requests/{id}",
-."/api/admin/projects/{projectId}/change-requests/{id}/comments",
-."/api/admin/projects/{projectId}/change-requests/{id}/state",
-."/api/admin/projects/{projectId}/change-requests/{id}/title",
-."/api/admin/projects/{projectId}/environments/{environment}/change-requests",
-."/api/admin/projects/{projectId}/environments/{environment}/change-requests/config"
-))' > modified-openapi.json
+# # Delete some schemas and paths using those schemas
+# cat openapi.json | \
+# jq 'del(.components.schemas | (
+#     .changeRequestsSchema,
+#     .changeRequestSchema,
+#     .changeRequestChangeSchema,
+#     .changeRequestSegmentChangeSchema,
+#     .changeRequestCreateSegmentSchema,
+#     .changeRequestCreateSchema,
+#     .changeRequestFeatureSchema,
+#     .changeRequestEventSchema,
+#     .changeRequestOneOrManyCreateSchema
+# ))' | \
+# jq 'del(.paths | (
+# ."/api/admin/features-batch/export",
+# ."/api/admin/projects/{projectId}/change-requests",
+# ."/api/admin/projects/{projectId}/change-requests/config",
+# ."/api/admin/projects/{projectId}/change-requests/open",
+# ."/api/admin/projects/{projectId}/change-requests/pending",
+# ."/api/admin/projects/{projectId}/change-requests/pending/{featureName}",
+# ."/api/admin/projects/{projectId}/change-requests/{changeRequestId}/changes/{changeId}",
+# ."/api/admin/projects/{projectId}/change-requests/{id}",
+# ."/api/admin/projects/{projectId}/change-requests/{id}/comments",
+# ."/api/admin/projects/{projectId}/change-requests/{id}/state",
+# ."/api/admin/projects/{projectId}/change-requests/{id}/title",
+# ."/api/admin/projects/{projectId}/environments/{environment}/change-requests",
+# ."/api/admin/projects/{projectId}/environments/{environment}/change-requests/config"
+# ))' > modified-openapi.json
 
 openapi-generator-cli generate \
     --git-user-id Unleash \
     --git-repo-id unleash-server-api-go \
-    --openapi-normalizer REMOVE_ANYOF_ONEOF_AND_KEEP_PROPERTIES_ONLY=true \
-    --openapi-normalizer SIMPLIFY_ONEOF_ANYOF=true \
-    --openapi-normalizer SIMPLIFY_ANYOF_STRING_AND_ENUM_STRING=true \
     --openapi-normalizer REFACTOR_ALLOF_WITH_PROPERTIES_ONLY=true \
+    --openapi-normalizer RESOLVE_INLINE_ENUMS=true \
+    --openapi-normalizer REF_AS_PARENT_IN_ALLOF=true \
```

Comment:
Not sure if these changes are needed

Reply:
Any idea what they actually do or are we at the point where we're fiddling with options and hoping for the best?

Reply:
Yep, makes sense. Right now if it works, I'm happy, mostly asking for context